### DISCUSSION TITLE
Some question about nvcc compile options

### DISCUSSION THREAD

**Discussion by MARD1NO** [NVIDIA/cutlass#1290] [Q&A] [answered]
Recently I want to integrate some cutlass operators into PyTorch, and I found in CUDAExtension we can pass some compile options. In other project it usually set: 
```python
"--ptxas-options=-O2",
"--ptxas-options=-allow-expensive-optimizations=true",
```

In my GEMM test, I found turn off these 2 options will get better performance, but there is no documentations about ptx. I just want to know what these options means, and hope cutlass develop team will give me some suggestions! :D

**Comment by hwu36** [comment, marked as answer]
when you compile cutlass, just use cutlass's nvcc flags.  that is what nvcc team uses to optimize cutlass.
